### PR TITLE
Use topojson input from node_modules/sane-topojson

### PR DESCRIPTION
**Pull request by archmoj**
Removing `topojson` files from `vendor` folder to avoid duplicates on (`dist` and `vendor`) and simplify build process by using raw files from `sane-topojson/dist` iwithin `node_modules` folder.

@plotly/plotly_js

### CODE REVIEW

File: devtools/test_dashboard/devtools.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 var Fuse = require('fuse.js/dist/fuse.common.js');
 var mocks = require('../../build/test_dashboard_mocks.json');
 var credentials = require('../../build/credentials.json');
+var constants = require('@src/plots/geo/constants');
```
Comment (alexcjohnson) [inline devtools/test_dashboard/devtools.js:8]: don't you want `tasks/util/constants`?

Comment (archmoj) [reply devtools/test_dashboard/devtools.js:8]: This is exactly the same as what is used in `geo_test`
https://github.com/plotly/plotly.js/blob/69ffb2c4a3fa6ed88d9cafd9db497741d8f24c23/test/jasmine/tests/geo_test.js#L6

Comment (archmoj) [reply devtools/test_dashboard/devtools.js:8]: You are right. We are not really using local files here and in `geo_test` on this branch.

Comment (archmoj) [reply devtools/test_dashboard/devtools.js]: Good catch! 
Fixed in https://github.com/plotly/plotly.js/pull/5492/commits/343faa898cdc92e1e5ceee821c369bb09a135c4a.

File: devtools/test_dashboard/devtools.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ var Tabs = {
         Plotly.setPlotConfig({
 
             // use local topojson files
-            topojsonURL: '../../vendor/topojson/',
+            topojsonURL: '/node_modules/sane-topojson/dist/',
```
Comment (archmoj) [inline devtools/test_dashboard/devtools.js:22]: Or it should be 
```js
topojsonURL: '../../node_modules/sane-topojson/dist/',
```
?

Comment (archmoj) [reply devtools/test_dashboard/devtools.js]: Addressed in https://github.com/plotly/plotly.js/pull/5492/commits/815d0471c8e3af473a21bfdfdd48b3162091b4b4.

**Review by alexcjohnson — APPROVED** [review]
💃 LGTM!

**Pull request merged**